backend/fake_private_key.py

```diff
@@ -0,0 +1,29 @@
+"""Used by the tests to generate fake JWT."""
+
+FAKE_PRIVATE_KEY = b"""-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Do we need the `.pem` file ? If yes can we load `backend/fake_private_key.pem` into this file instead of needing to copy paste them ?

Reply:
We don't need the PEM file. Let's just use the PY file.